assessments/guide/security-reviewer.md

```diff
@@ -0,0 +1,31 @@
+# Security Reviewers
+
+Reviewers will try to understand the
```

Comment:
I recommend we have a minimum number of reviewers to ensure diverse and possibly even conflicting opinions, lest there is too much dependence on one or two reviewers?  it also encourages expansion of the network of reviewers so we cultivate a large, diverse experience base.

Reply:
specified in outline.md " assigned to lead [security reviewer](security-reviewer.md) who
   will recruit a second reviewer"